http_server/sync_handler.pony

```diff
@@ -1,6 +1,7 @@
-use "valbytes"
 use "debug"
 
+use "valbytes"
+
```

Comment:
This is unrelated at this point, no new uses were added so I think this should be rolled back